docs/en/observability/monitor-k8s/monitor-k8s-logs.asciidoc

```diff
@@ -0,0 +1,357 @@
+[discrete]
+[[monitor-kubernetes-logs]]
+== Part 1: Monitor logs
+
+Collecting and analyzing logs of both Kubernetes Core components and various
+applications running on top of Kubernetes is a powerful tool for Kubernetes
+observability. Containers running within Kubernetes pods produce logs as stdout
+or stderr. These logs are written to a location known to Kubelet.
+
+To collect pod logs, all you need is {filebeat} running as a DaemonSet
+in your Kubernetes cluster. You configure {filebeat} to communicate with the
+Kubernetes API server, get the list of pods running on the current host, and
+collect the logs the pods are producing. Those logs are annotated with all the
+relevant Kubernetes metadata, such as pod ID, container name, container labels
+and annotations, and so on.
+
+[discrete]
+=== Deploy {filebeat} to collect logs
+
+To start collecting logs, deploy and run an instance of {filebeat} on each
+Kubernetes host. {filebeat} communicates with the Kubernetes API server to
+retrieve information about the pods running on the host, all the metadata
+annotations, and the location of the log files.
```

Comment:
Log location is not know to Filebeat via k8s API. It is defined in the container input section, for instance at https://github.com/elastic/beats/blob/master/deploy/kubernetes/filebeat-kubernetes.yaml#L14

Reply:
Right, I must have done a bad edit at some point. :-) I think I meant that it uses annotations to decide which logging modules to apply. But looking a this now, I'm not sure the detail belongs here.